xbmc/addons/PVRClient.cpp

```diff
@@ -1618,77 +1592,6 @@ time_t CPVRClient::GetBufferTimeEnd(void) const
   return time;
 }
 
-bool CPVRClient::CanAutoconfigure(void) const
```

Comment:
Yep, it is. I did not remove this feature but it is a matter of fact that this is dead code. AFAIK it never worked reliably, btw. Have seen many complaints in the forum in the past.